src/bio/bio_xstream.c

```diff
@@ -308,7 +308,8 @@ bio_nvme_init(const char *nvme_conf, int numa_node, unsigned int mem_size,
 	}
 
 	nvme_glb.bd_mem_size = mem_size;
-	D_STRNDUP(nvme_glb.bd_nvme_conf, nvme_conf, strlen(nvme_conf));
+	if (nvme_conf)
```

Comment:
we should probably do all the config input parameter checking in one place, I will push small PR to do this

Reply:
sure, thanks.